torch/csrc/jit/codegen/cuda/lower_utils.cpp

```diff
@@ -109,6 +110,25 @@ bool isTvOp(const Expr* expr) {
   return false;
 }
 
+bool isLdMatrixOp(const Expr* expr) {
+  if (auto ldst = dynamic_cast<const LoadStoreOp*>(expr)) {
+    return ldst->opType() == LoadStoreOpType::LdMatrix ||
+        ldst->opType() == LoadStoreOpType::LdMatrixTranspose;
+  }
+  return false;
+}
+
+bool isTensorScalarFillOp(const Expr* expr) {
+  if (auto uop = dynamic_cast<const UnaryOp*>(expr)) {
```

Comment:
Don't we want to include `LoadStoreOp` as well? I don't think it can have a scalar input at this point, but I think future updates might include other types of loads and stores with a scalar input.

Reply:
Ah good idea. Yes we do have use cases like that already. I'll update.

Reply:
Added. Thanks!